NEWS.md

```diff
@@ -1,4 +1,10 @@
-# epiprocess 0.7.0.9000
+# epiprocess 0.7.1.9999
+
+* Patch `select` for grouped `epi_df`s PR #390
+* switched `other_keys` default from `NULL` to `character(0)` PR #390
+* minor doc updates PR #393
+
+# epiprocess 0.7.0.9999
```

Comment:
We've been using 0.7.0.9999 to mean "development past 0.7.0 that we haven't assigned a real version number yet" & bumping to e.g., 0.7.1 or 0.8.0 on release.  Sound all right?  If so maybe merge with 0.7.0.9999?

Reply:
We're switching to [this version numbering scheme](https://delphi-org.slack.com/archives/C028AUR6MV1/p1705606092907929) so 0.7.1.9999 is good.